Calculator.Lonchanick/Calculator.Lonchanick/Calculator.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Diagnostics;
+using Newtonsoft.Json;
+
+namespace Calculator.Lonchanick
+{
+    public class Calculator
+    {
+        JsonWriter writer;
+        public Calculator() {
+
+            StreamWriter logFile = File.CreateText("calculatorlog.json");
+            logFile.AutoFlush = true;
+            writer = new JsonTextWriter(logFile);
+            writer.Formatting = Formatting.Indented;
+            writer.WriteStartObject();
+            writer.WritePropertyName("Operations");
+            writer.WriteStartArray();
+        }
+        public void Finish()
+        {
+            writer.WriteEndArray();
+            writer.WriteEndObject();
+            writer.Close();
+        }
+        public void DoOperation(double num1, double num2, string op)
+        {
+            
+            double result = double.NaN; // Default value is "not-a-number" if an operation, such as division, could result in an error.
+            writer.WriteStartObject();
+            writer.WritePropertyName("Operand1");
+            writer.WriteValue(num1);
+            writer.WritePropertyName("Operand2");
+            writer.WriteValue(num2);
+            writer.WritePropertyName("Operation");
+            // Use a switch statement to do the math.
+            switch (op)
+            {
+                case "a":
+                    result = num1 + num2;
+                    writer.WriteValue("Add");
+                    break;
+                case "s":
+                    result = num1 - num2;
```

Comment:
At the moment the result of the operations isn't being printed to the user. Was that your intent?

![image](https://user-images.githubusercontent.com/108711476/210304277-efca758e-57e1-4085-ac63-92f9fb8602bf.png)

Reply:
Fixed and new functionalities added